src/main/java/seedu/programmer/storage/JsonAdaptedLab.java

```diff
@@ -30,7 +30,7 @@ public JsonAdaptedLab(@JsonProperty("labNumValue") String labNumValue,
      */
     public JsonAdaptedLab(Lab lab) {
         requireNonNull(lab);
-        this.labNumValue = lab.getLabNumValue();
+        this.labNumValue = String.valueOf(lab.getLabNumValue());
         this.actualScoreValue = lab.getLabResultValue();
         this.totalScoreValue = lab.getLabTotalValue();
```

Comment:
It might be better if we consistently use get..Value() for JsonAdaptedLab?

Reply:
Addressed in previous comment